src/appleseed/renderer/kernel/aov/aovaccumulator.cpp

```diff
@@ -89,16 +88,12 @@ BeautyAOVAccumulator::BeautyAOVAccumulator()
 
 void BeautyAOVAccumulator::set(const Spectrum& value)
 {
-    m_color_space = value.is_rgb() ? ColorSpaceLinearRGB : ColorSpaceSpectral;
-    m_color = value;
+    m_color = value.to_rgb(g_std_lighting_conditions);
```

Comment:
Can we make the lighting conditions a thread local variable too?
In our color management PR, lighting conditions are selected as part of the working color space selection and g_std_lighting_conditions does not exist anymore.